.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install poetry
+        poetry install
+    - name: Lint check
+      run: poetry run black --check .
+    - name: Tests
+      run: poetry run pytest
+  publish:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install poetry
+        poetry install
+    - name: Build package
+      run: poetry build
+    - name: Publish package
+      run: |
+        poetry publish -u ${{ secrets.PYPI_USER }} -p ${{ secrets.PYPI_PASS }} --no-interaction
```

We're going to publish packages with [API token](https://pypi.org/help/#apitoken):
```suggestion
        poetry publish -u "__token__" -p ${{ secrets.PYPI_TOKEN }} --no-interaction
```